core/src/Kokkos_HIP_Space.hpp

```diff
@@ -657,7 +657,7 @@ class HIP {
 
   ~HIP() = default;
   HIP();
-  //  explicit HIP( const int instance_id );
+  HIP(hipStream_t stream);
```

Comment:
I didn't drop anything this is a new constructor that we did not have before. Since the new constructor uses an `hipStream_t` instead of an `int` I don't see the point of using `explicit`

Reply:
Overlook on my part.  Sorry :)